app/views/users/index.html..erb

```diff
@@ -0,0 +1,10 @@
+<% provide(:title, t(:all_users)) %>
```

Comment:
This file is named `app/views/users/index.html.erb` instead of `app/views/users/index.html..erb`, you are mistaking for the redundant `.` character